### PR TITLE
Use strfry and blastr for sending

### PR DESCRIPTION
**Pull request by ekzyis**

### CODE REVIEW

File: mutiny-core/src/nostr/mod.rs [hunk 1]
```diff
@@ -122,7 +122,7 @@ impl<S: MutinyStorage> NostrManager<S> {
         relays.push("wss://relay.primal.net".to_string());
         relays.push("wss://relay.damus.io".to_string());
 
-        // add blastr for default sending
+        // add strfry for default sending
         relays.push("wss://relay.mutinywallet.com".to_string());
```
Comment (benthecarman) [inline mutiny-core/src/nostr/mod.rs:126]: tbh we should add blastr back, can you do that

Comment (ekzyis) [reply mutiny-core/src/nostr/mod.rs:126]: you mean to have both relays? yes, can do

Comment (ekzyis) [reply mutiny-core/src/nostr/mod.rs:126]: you mean to have both relays? yes, can do

**Review by benthecarman — APPROVED** [review]

**Review by TonyGiorgio — APPROVED** [review]
awesome, thank you!

**Pull request merged**